### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by CrickSoldier**
I have done some changes in Prerequisites section to make it better for the viewer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 A collection of random configuration scripts to help with setting up a new Mac. This is also a list of apps and tools that I like to use.
 
 ## _Prerequisites_
-[Homebrew](http://brew.sh/)
+The it needed is [Homebrew](http://brew.sh/),you can install the homebrew by coping and pasting the following command in terminal.
```
Comment (voidrender) [inline README.md]: How about this instead?

```
[Homebrew](http://brew.sh/) is required to run these scripts. You can install Homebrew by copying and pasting the following command into your terminal.
```

**Review by voidrender — CHANGES_REQUESTED** [review]
Thanks! I like this idea, but would like to request a slight change to the wording.

**Comment by CrickSoldier** [comment]
I had changed the file as per the suggestion.

**Review by voidrender — APPROVED** [review]
Thanks!

**Pull request merged**